src/librustc_data_structures/work_queue.rs

```diff
@@ -53,7 +53,7 @@ impl<T: Idx> WorkQueue<T> {
         }
     }
 
-    /// Attempt to enqueue `element` in the work queue. Returns false if it was already present.
+    /// Attempt to pop element from the work queue.
```

Comment:
Nit: "an element".

Reply:
Repushed with change.

Change is in separate commit.  Let me know if I should squash it with rebase instead.

Reply:
Thanks — if you could squash instead, that would be great!